contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -703,7 +703,8 @@ def feerates(self, style, urgent=None, normal=None, slow=None):
 
     def fundchannel(self, node_id, amount, feerate=None, announce=True,
                     minconf=None, utxos=None, push_msat=None, close_to=None,
-                    request_amt=None, compact_lease=None):
+                    request_amt=None, compact_lease=None,
+                    mindepth: Optional[int] = None):
```

Comment:
Note that Changelog-Added line is wrong:

1. : should go before JSON-RPC otherwise I'm not sure we'll parse it correctly?
2. It's called "mindepth" not "minconf".

(BTW, I agree mindepth is the right name: prefer spec language, and it's "minimum_depth" there.)

Reply:
Good catch, mentally I sometimes made the switch between `minconf` (confirmations required for funds to be considered when funding) and `mindepth` (depth required to consider a funded channel active), and I'm sure I won't be the only one, but agreed that it is more in line with the spec nomenclature.